recipe/build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-mkdir build
-cd build
+mkdir build_release
+cd build_release
```

Comment:
Yes. gflags includes a bazel BUILD file. Windows is case insensitive so the name conflicted.